packages/react-button/src/components/SplitButton/useSplitButtonStyles.ts

```diff
@@ -0,0 +1,127 @@
+import { makeStyles, mergeClasses } from '@fluentui/react-make-styles';
+import type { SplitButtonState } from './SplitButton.types';
+
+const SplitButtonClassNames = {
+  button: 'SplitButton-button',
+  menuButton: 'SplitButton-menuButton',
+};
+
+const useStyles = makeStyles({
+  root: theme => ({
+    display: 'inline-flex',
```

Comment:
Does either the button or menu button need specific flex properties (initial, shrink ,grow)?

Reply:
We are mostly using this to be able to use `alignItems` and `justifyContent` to properly align everything here.